moto/ec2/models.py

```diff
@@ -407,6 +435,17 @@ def __init__(self, ec2_backend, image_id, user_data, security_groups, **kwargs):
         self.disable_api_termination = kwargs.get("disable_api_termination", False)
         self._spot_fleet_id = kwargs.get("spot_fleet_id", None)
         associate_public_ip = kwargs.get("associate_public_ip", False)
+
+        # Create VM instance
+        subprocess.run("vboxmanage clonevm --name instance_{} --register base"
```

Comment:
The user of Instance class has to be able to define does he want `vbox` or pure mocked instance.